apis_core/utils/rdf2.py

```diff
@@ -0,0 +1,86 @@
+# SPDX-FileCopyrightText: 2025 Birger Schacht
+# SPDX-License-Identifier: MIT
+
+import logging
+from collections import defaultdict
+from pathlib import Path
+
+import tomllib
+from AcdhArcheAssets.uri_norm_rules import get_normalized_uri
+from django.apps import apps
+from rdflib import RDF, BNode, Graph
+
+logger = logging.getLogger(__name__)
+
+
+def find_matching_config(graph: Graph) -> dict | None:
+    models_with_config = [
+        model for model in apps.get_models() if hasattr(model, "rdf_configs")
+    ]
+    for model in models_with_config:
+        for path in model.rdf_configs():
+            config = tomllib.loads(Path(path).read_text())
+            for _filter in config.get("filters", []):
+                try:
+                    triples = [
+                        (
+                            None,
+                            graph.namespace_manager.expand_curie(predicate),
+                            graph.namespace_manager.expand_curie(obj),
+                        )
+                        for predicate, obj in _filter.items()
+                    ]
+                    triples = [triple in graph for triple in triples]
+                    if all(triples):
+                        logger.debug("Using %s for parsing graph", path)
+                        config["model"] = model
+                        return config
+                except ValueError:
+                    pass
```

Comment:
```suggestion
                except ValueError as e:
                    logger.debug(f"Parsing values with filter {_filter} failed: {e}")
```
add some debugging info when parsing failes